packages/opentelemetry/src/index.ts

```diff
@@ -291,5 +340,30 @@ export async function instrumented<T>(
     });
 }
 
-export { trace, context, SpanStatusCode } from '@opentelemetry/api';
+export async function instrumentedWithMetrics<T>(
+  meter: Meter,
+  name: string,
+  fn: () => Promise<T> | T
+): Promise<T> {
+  const success = meter.createCounter(`${name}.success`, { valueType: ValueType.INT });
+  const error = meter.createCounter(`${name}.error`, { valueType: ValueType.INT });
+  const histogram = meter.createHistogram(`${name}.duration`, {
+    unit: 'milliseconds',
+    valueType: ValueType.DOUBLE,
+  });
+
+  const start = Date.now();
```

Comment:
Should we be using [`performance.now()`](https://nodejs.org/api/perf_hooks.html#performancenow) instead?

Reply:
Yes, I keep forgetting this exists! Will change.

Reply:
Switched to `performance.now()` in af91b32.